components/sql/src/main/scala/pl/touk/nussknacker/sql/service/DatabaseEnricherInvoker.scala

```diff
@@ -30,16 +30,15 @@ class DatabaseEnricherInvoker(query: String,
   override def invokeService(params: Map[String, Any])
                             (implicit ec: ExecutionContext, collector: ServiceInvocationCollector, contextId: ContextId, componentUseCase: ComponentUseCase): Future[queryExecutor.QueryResult] =
     getTimeMeasurement().measuring {
-      Future.successful {
-        queryDatabase(queryArgumentsExtractor(argsCount, params))
-      }
+      queryDatabase(queryArgumentsExtractor(argsCount, params))
     }
 
-  protected def queryDatabase(queryArguments: QueryArguments): queryExecutor.QueryResult =
+  protected def queryDatabase(queryArguments: QueryArguments)(implicit ec: ExecutionContext): Future[queryExecutor.QueryResult] = Future {
```

Comment:
Can you move this `Future {` to separate line? I don't know if only I have that, but for me, this Future vanish on the end of line.